thereadingmachine/scraper/news_scraper/pipelines.py

```diff
@@ -69,35 +61,33 @@ def process_item(self, item, spider):
             raise DropItem('Empty Article in %s' % item)
 
 
-class AmisJsonPipeline(object):
+class AmisScrapePipeline(object):
 
     def __init__(self):
-        self.datafiles = {}
+        self.data_list = []
         self.today = datetime.today()
 
     def open_spider(self, spider):
```

Comment:
you can leave the method not implemented instead of cluttering the class

Reply:
Done